### ISSUE TITLE
Incomplete Gamma Function

### ISSUE THREAD

**Issue by gideonsimpson** [JuliaMath/SpecialFunctions.jl#115]
Maybe it's there and I'm not seeing it, but is there a way straightforward way to evaluate the incomplete gamma function?

**Comment by simonbyrne** [comment]
For historical reasons, the incomplete gamma is in https://github.com/JuliaStats/StatsFuns.jl (though we should probably merge them at some point).

**Comment by AshtonSBradley** [comment]
You can also call high quality implementations of gamma and many related functions wrapped by GSL.jl:
https://github.com/JuliaMath/GSL.jl
This works great on julia 0.6, but isn't updated to work on julia 1.0 yet. Not sure what the status is, but there seems to be some issue with the update https://github.com/JuliaMath/GSL.jl/pull/71 
@jiahao @rdeits @jagot is there anything you can tell us about the status of this PR? Is there a way forward for this essential library? Any way wider community can help get it to 1.0?

**Comment by rdeits** [comment]
That PR gets all but one test passing on Julia v0.7 (or at least it did back in August when I wrote it). I'm not a maintainer of the package, so there's not much I can do to move it forward, unfortunately.

As a general note, you can always install a package from its exact git URL and commit name, so you should be able to install the code used in that PR with: 

```
pkg> add https://github.com/rdeits/GSL.jl#e1da596
```

**Comment by AshtonSBradley** [comment]
I think the tagged release is just waiting on Windows CI https://github.com/JuliaMath/GSL.jl/pull/79

**Comment by JohnCortese** [comment]
Hi Folks,

There is an error in the Incomplete Gamma function in the Julia Special Functions package. The Incomplete gamma function with a=1/2 yields values in error around x=3.0. Note that with this value of a=1/2, the Incomplete error function is proportional to the complementary error function and should be smooth around x=3.0. Please see the attached pdf plot as well as the Julia code and a log file of the Julia 1.60 run that generated the plot. 

Any comments or recommendations are welcome. Thank you !

I have never posted about a Julia error nor used GitHub, so if I am posting about this error in the wrong place please let me know and my apologies !


[IncompleteGammaError.txt](https://github.com/JuliaMath/SpecialFunctions.jl/files/6253190/IncompleteGammaError.txt)
[IncGammaError.pdf](https://github.com/JuliaMath/SpecialFunctions.jl/files/6253193/IncGammaError.pdf)
[ErrorLog.txt](https://github.com/JuliaMath/SpecialFunctions.jl/files/6253195/ErrorLog.txt)

**Comment by giordano** [comment]
@JohnCortese looks like the same issue as #303?

**Comment by JohnCortese** [comment]
Hi Giordano,

Yes, the issue I wrote about earlier today is identical to # 303. Sorry - I am new to Julia error reporting and GitHub. 

John

**Comment by giordano** [comment]
No worries :slightly_smiling_face:

**Comment by stevengj** [comment]
Closing, since the original issue is fixed, and the remaining issue is a duplicate of #303.

**Comment by JohnCortese** [comment]
Thank you Steven. The issue remains in Julia 1.6, which is the most recent version. Will the fix to Gamma be incorporated in the next Julia update ?

**Comment by stevengj** [comment]
The SpecialFunctions package is separate from Julia releases. We can tag a new version as soon as we track down the bug
